sdks/python/test-suites/tox/py37/build.gradle

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Unit tests for Python 3.7
+ */
+
+plugins { id 'org.apache.beam.module' }
+applyPythonNature()
+
+// Required to setup a Python 3 virtualenv.
+project.ext.python3 = true
```

Comment:
This is changed recently to have ability to configure a particular python version in virtualenv. Should be `pythonVersion = '3.7'`. See [the example](https://github.com/apache/beam/blob/master/sdks/python/test-suites/tox/py36/build.gradle#L27).

Reply:
Thanks! I started on this before your PR was merged. It's fixed now.